src/mds/Server.cc

```diff
@@ -244,14 +245,14 @@ void Server::dispatch(const cref_t<Message> &m)
     handle_client_reconnect(ref_cast<MClientReconnect>(m));
     return;
   }
-
+  bool sessionclosed_isok = replay_unsafe_with_closed_session;
```

Comment:
Add the comment above this line explaining the scenario you described here https://github.com/ceph/ceph/pull/29059#issuecomment-561948489

Reply:
> Add the comment above this line explaining the scenario you described here [#29059 (comment)](https://github.com/ceph/ceph/pull/29059#issuecomment-561948489)

done